src/OrchardCore.Modules/OrchardCore.Features/AdminMenu.cs

```diff
@@ -22,8 +22,22 @@ public AdminMenu(IStringLocalizer<AdminMenu> stringLocalizer)
 
     protected override ValueTask BuildAsync(NavigationBuilder builder)
     {
+        if (NavigationHelper.UseLegacyFormat())
+        {
+            builder
+                .Add(S["Configuration"], configuration => configuration
+                    .Add(S["Features"], S["Features"].PrefixPosition(), deployment => deployment
+                        .Action("Features", "Admin", _routeValues)
+                        .Permission(Permissions.ManageFeatures)
+                        .LocalNav()
+                    )
+                );
+
+            return ValueTask.CompletedTask;
+        }
+
         builder
-            .Add(S["Configuration"], configuration => configuration
+            .Add(S["Tools"], tools => tools
```

Comment:
Are you suggesting to move Features and Admin Menu to settings?

Reply:
I strongly disagree with your suggestion to moving Feature and Admin Menu to the Settings menu. I think they should stay under tools. These are better fit for tools. + we don't need to treat everything are settings and clutter the settings menu more.

Reply:
Why is toggling feature states more of a tool than configuration?

Reply:
It's a tool to enable/disable features. It's not purely settings.  We should not just start moving things into settings just because they make changes. We should only put settings related items under settings.

Reply:
Isn't the configuration page to change the site's title also but a tool to set the title? Or would a different features UI, like with a list of checkboxes for each feature and a Save button at the bottom, make it more settings-y? :)

Reply:
To me, if we have a form to fill out or change settings, that would be under the Settings. Features, its a tool to manage features.

Reply:
What if I tell you that the Features page is actually a form already? :)

My point is, we have to distinguish between these based on the underlying principle and not a perception of the current UI, and toggling feature states is changing configuration, i.e. settings, in the same way as changing the site title is.

Reply:
If you strongly feel it should be in Settings while I strongly feel it should be in Tools, I think we should get other opinions on this. We'll discuss it more in the Steering meeting today.

Reply:
@Piedone talked about this in the meeting. @sebastienros  and @gvkries suggested to keep it in Tools as well.